### PR TITLE
[zh_CN]translation style_guild

### PR DESCRIPTION
**Pull request by sc1000y**
Signed-off-by: sc1000y <8299176+sc1000y@users.noreply.github.com>

translation style_guild #313

### CODE REVIEW

**Comment by sc1000y** [comment]
#313

File: docs/locale/zh_CN/source/style_guide.md [hunk 1]
```diff
@@ -1,136 +1,141 @@
...
+**如果有疑问，请使用文档本身来指导如何修改内容格式**
 
-* [For RST formatting](http://hyperledger-fabric.readthedocs.io/en/release-1.4/channel_update_tutorial.html).
+* [RST格式](http://hyperledger-fabric.readthedocs.io/en/release-1.4/channel_update_tutorial.html).
 
-* [For Markdown formatting](http://hyperledger-fabric.readthedocs.io/en/release-1.4/peers/peers.html).
+* [Markdown格式](http://hyperledger-fabric.readthedocs.io/en/release-1.4/peers/peers.html).
 
-If you just want to look at how things are formatted, you can navigate to the Fabric repo to look at the raw file by clicking on `Edit on Github` link in the upper right hand corner of the page. Then click the `Raw` tab. This will show you the formatting of the doc. **Do not attempt to edit the file on Github.** If you want to make a change, clone the repo and follow the instructions in [Contributing](./CONTRIBUTING.html) for creating pull requests.
+如果您只想查看内容的格式，可以通过单击页面右上角的`Edit on Github`链接导航到Fabric repo查看原始文件。然后单击`Raw`选项卡。这将显示文档的格式。**不要试图在Github上编辑文件。**如果要进行更改，请克隆repo并按照[贡献]（./CONTRIBUTING.html)用于创建请求请求。
```
Comment (stone-ch) [inline docs/locale/zh_CN/source/style_guide.md]: 请克隆仓库并按照[贡献]（./CONTRIBUTING.html)来创建PR。

File: docs/locale/zh_CN/source/style_guide.md [hunk 2]
```diff
@@ -1,136 +1,141 @@
...
 
-## Word choices
+## 文字选择
 
-**Avoid the use of the words "whitelist", "blacklist", "master", or "slave".**
+**避免使用 "whitelist", "blacklist", "master", or "slave".**
```
Comment (stone-ch) [inline docs/locale/zh_CN/source/style_guide.md]: 这几个单词，翻译成中文吧

File: docs/locale/zh_CN/source/style_guide.md [hunk 3]
```diff
@@ -1,136 +1,141 @@
...
 
-**Is it ever appropriate to use a dash?**
+**使用破折号是否合适?**
 
-Dashes can be incredibly useful but they're not necessarily as technically appropriate as using separate declarative sentences. Let's consider this example sentence:
+破折号可能非常有用，但在原则上不一定合适于分割陈述句。让我们来看看这个例句:
 
 ```
-This leaves us with a trimmed down JSON object --- config.json, located in the fabric-samples folder inside first-network --- which will serve as the baseline for our config update.
+这给我们留下了一个精简的JSON对象---config.json，位于first network内的fabric samples文件夹中---它将作为配置更新的基线。
```
Comment (stone-ch) [inline docs/locale/zh_CN/source/style_guide.md]: ---config.json 缺了一个空格应该为 --- config.json

**Comment by davidkhala** [comment]
@sc1000y Would you mind squash into single commits?

**Pull request closed without merging**